app/src/main/res/layout/drawer_list_item.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+    <data>
+        <import type="android.view.View"/>
+        <variable
+            name="menu"
+            type="org.fossasia.openevent.app.main.menu.OpevMenu"/>
+    </data>
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <LinearLayout
+            android:id="@+id/menu_separator"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/spacing_normal"
+            android:gravity="center_vertical"
+            android:visibility="@{menu.belowSeparator ? View.VISIBLE : View.GONE}">
+
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="1dp"
+                android:background="#e4e4e4"/>
+
+        </LinearLayout>
```

Comment:
actually section divider has to be 16dp thick with line at middle. Thats why this. FrameLayout wil just replace LinearLayout here.